packages/main/src/components/SplitterLayout/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { SplitterElementPropTypes } from '@ui5/webcomponents-react/dist/SplitterElement';
+import { SplitterLayoutContext } from '@ui5/webcomponents-react/dist/SplitterLayoutContext';
+import React, { CSSProperties, forwardRef, ReactElement, Ref } from 'react';
+import { createUseStyles } from 'react-jss';
+import { CommonProps } from '../../interfaces/CommonProps';
+import { styles } from './SplitterLayout.jss';
+import '@ui5/webcomponents-icons/dist/vertical-grip.js';
+import '@ui5/webcomponents-icons/dist/horizontal-grip.js';
+import { useConcatSplitterElements } from './useConcatSplitterElements';
+import clsx from 'clsx';
+
+const useStyles = createUseStyles(styles, { name: 'SplitterLayout' });
+
+export interface SplitterLayoutPropTypes extends CommonProps {
+  /**
+   * Controls the width of the `SplitterLayout` container.<br />
+   */
+  width?: string;
+  /**
+   * Controls the height of the `SplitterLayout` container.<br />
+   */
+  height?: string;
+  /**
+   * Controls if the `SplitterLayout` is displayed `horizontal` or `vertical`.<br />
+   */
+  orientation?: 'horizontal' | 'vertical';
```

Comment:
Please either create an enum for the `orientation` similar to [sap.ui.core.Orientation](https://ui5.sap.com/#/api/sap.ui.core.Orientation) or replace it with a boolean (`horizontal?: boolean;`), because there are only two values anyways. I would prefer option two, but I'll leave it up to you to decide what you think is best here :)